src/main/java/com/mojang/datafixers/types/Type.java

```diff
@@ -163,29 +164,25 @@ public TypeTemplate template() {
         // We can not use computeIfAbsent because this is a recursive call that will block server startup
         // during the Bootstrap phrase that's trying to pre cache these rewrites.
         final Optional<? extends RewriteResult<?, ?>> rewrite = REWRITE_CACHE.get(key);
-        //noinspection OptionalAssignedToNull
```

Comment:
FYI this triggered a useless warning in the IDE is why I had added that, but if you don't want it, up to you.